site/en/docs/privacy-sandbox/attribution-reporting/report-schedules/index.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Report schedules'
+subhead: >
+  Learn when the Attribution Reporting API sends aggregatable and event-level reports.
+description: >
+  Learn when the Attribution Reporting API sends aggregatable and event-level reports.
+date: 2023-03-06
+updated: 2023-07-21
+authors:
+  - maudn
+---
+
+With the Attribution Reporting API, reports are not sent immediately after a user converts. Instead, the browser sends them with a delay.
+
+Note that regardless of report type, reports are only sent when the browser is running and online. After the first failure to send, the browser tries to send the report again after 5 minutes. After the second failure, the browser tries to send the report again after 15 minutes. After that, it's not sent.
+
```

Comment:
Less of a writing question, more of a tech one—what's the solution to the problem? Regenerate a report when the sender is back online?

Reply:
Great question. You can't generate a report, only the browser can, as far as I know. @maudnals what happens to the data in the report that is not sent?